src/components/LikedByWho.tsx

```diff
@@ -0,0 +1,38 @@
+import { st, classes } from '../style/reply.st.css';
+import { classes as commentClassesNames } from '../style/comments.st.css';
+import { classes as mainClassesNames } from '../style/main.st.css';
+import { FIRST_LIKED_BY_PROFILE_PICTURE, SECOND_LIKED_BY_PROFILE_PICTURE } from '../constants';
+
```

Comment:
In the props for this component you should receive an array of strings representing the people who liked the photo
you can also receive an array with the srcs of the images to display

Reply:
add comments on the parent and moved it up

Reply:
"like"